docs/src/components/Layout/GlobalNav/components/icons/IconLink.tsx

```diff
@@ -0,0 +1,29 @@
+import { Icon } from '@aws-amplify/ui-react';
+
+export enum NavMenuIconType {
+  TWITTER = 'TWITTER',
+  DISCORD = 'DISCORD',
+}
+
+export function IconLink({ iconType }: { iconType: NavMenuIconType | string }) {
+  switch (iconType) {
+    case NavMenuIconType.TWITTER:
+      return (
+        <Icon
+          ariaLabel="Twitter"
+          viewBox={{ width: 20, height: 18 }}
+          pathData="M20 2.79906C19.2641 3.12541 18.4733 3.34598 17.6433 3.44517C18.4905 2.93733 19.1411 2.1332 19.4475 1.17508C18.6547 1.64525 17.7765 1.98675 16.8419 2.1708C16.0933 1.37334 15.027 0.875 13.8468 0.875C11.5807 0.875 9.74344 2.71216 9.74344 4.97815C9.74344 5.29974 9.7798 5.61292 9.84974 5.91322C6.43952 5.74211 3.41607 4.10855 1.3923 1.62609C1.0391 2.23211 0.836753 2.93696 0.836753 3.68893C0.836753 5.11246 1.56121 6.3684 2.6622 7.1042C1.98954 7.08291 1.35689 6.89834 0.803614 6.59101C0.803321 6.60813 0.803321 6.62532 0.803321 6.64258C0.803321 8.63066 2.21771 10.2891 4.09481 10.666C3.75047 10.7598 3.38798 10.8099 3.01372 10.8099C2.74933 10.8099 2.49226 10.7842 2.24178 10.7363C2.76389 12.3665 4.27923 13.5528 6.07476 13.5859C4.67047 14.6864 2.9012 15.3424 0.978821 15.3424C0.647646 15.3424 0.321007 15.323 0 15.2851C1.81587 16.4493 3.97271 17.1286 6.28992 17.1286C13.8372 17.1286 17.9644 10.8762 17.9644 5.45395C17.9644 5.27604 17.9604 5.09908 17.9525 4.92314C18.7542 4.34462 19.4499 3.62192 20 2.79906Z"
+        />
+      );
+    case NavMenuIconType.DISCORD:
+      return (
+        <Icon
+          ariaLabel="Discord"
+          viewBox={{ width: 20, height: 16 }}
```

Comment:
@0618 , is it possible to set the `viewBox` with CSS?

I think originally it was set to different heights because the twitter icon was a little bit larger than the discord icon. I can set it to 16 for everything though

Reply:
So it's better to have a standard of size of all the icons we use, then we can use a `map` here.

as for `viewBox`, we can a constant like

```ts
const iconViewBox = { width: 20, height: 16 }
```